### ISSUE TITLE
Upgrade to 4.0.4

### ISSUE THREAD

**Issue by nateberkopec** [mojodna/heroku-buildpack-jemalloc#2]

**Comment by nateberkopec** [comment]
Also, it'd be nice if we could download jemalloc from some more official source. 

EDIT: You could probably add it to the git repo here and use something like rawgit.com to download from it.

**Comment by sethfitz** [comment]
Agreed, though it needs to be built at some point. Adding it to the git repo would make it available at build time (w/o the download), but would also bloat the repo some, slowing things down.

Attaching it to a release might actually do the trick. I'll try creating a release for the current version and adding the binary.

(I don't actually use this buildpack--it was to facilitate some experiments a while back, so if you'd like to potentially adopt it, let me know.)

**Comment by sethfitz** [comment]
Yup, looks like that would work: https://github.com/mojodna/heroku-buildpack-jemalloc/releases/tag/v3.6.0

**Comment by nateberkopec** [comment]
@mojodna perhaps could adopt it, let me work a little more with it first. but it works great in production with the new buildpacks features, so I think more people should be looking at this.

**Comment by sethfitz** [comment]
Sweet, keep me posted!

**Comment by heaven** [comment]
Hi, any luck with building this? 4.2.1 is already available.

**Comment by nateberkopec** [comment]
Simply haven't had time - everything you need in the repo is there if you want to give it a shot! Will accept PRs.

**Comment by brian-kephart** [comment]
v5.0.0 just came out today. I'd do a PR for it, but I don't know how the build process works when building for a different environment than my local machine. How has this been done in the past?

**Comment by sethfitz** [comment]
There's a `Makefile` in the root that uses Docker to build it against the Heroku image (and extract it so that it can be uploaded to GitHub for referencing by `bin/compile`).

**Comment by sethfitz** [comment]
https://github.com/mojodna/heroku-buildpack-jemalloc/commit/0938c22 is largely suitable for a PR (without the line I commented on).
